modmesh/app/euler1d.py

```diff
@@ -32,34 +32,39 @@
     FigureCanvas, NavigationToolbar2QT as NavigationToolbar)
 from matplotlib.figure import Figure
 
+import modmesh as mm
 from .. import view
 from ..onedim import euler1d
 
 
 def load_app():
-    cmd = "win = mm.app.euler1d.run(animate=True, interval=10, max_steps=50)"
-    cmd += "; win.start()"
+    cmd = "plt = mm.app.euler1d.run(animate=True, interval=10, max_steps=50)"
+    cmd += "; plt.start()"
     view.app.pycon.command = cmd
 
 
-class ApplicationWindow(QtWidgets.QMainWindow):
-    def __init__(self, shocktube, max_steps):
+class Plotter:
+    def __init__(self, shocktube, max_steps, use_sub=None):
         if None is shocktube.gamma:
             raise ValueError("shocktube does not have constant built")
         if None is shocktube.svr:
             raise ValueError("shocktube does not have numerical solver built")
 
+        self.use_sub = mm.Toggle.USE_PYSIDE if use_sub is None else use_sub
```

Comment:
Make the minimal modification to make the app work with both C++ Qt sub window or PySide standalone window.